routes/article.js

```diff
@@ -1,140 +1,131 @@
-'use strict'
-
 const express = require('express')
-const bodyParser = require('body-parser')
-const Article = require('../models/ArticleModel.js')
-const Keyword = require('../models/KeywordModel.js')
+const Article = require('../models/ArticleModel')
+const Keyword = require('../models/KeywordModel')
 
 const router = express.Router()
 
+function getNextArticles(findParams, sortParams) {
+  return new Promise((resolve, reject) => {
+    Article.find(findParams)
+      .sort(sortParams)
```

Comment:
Promise style change: Call a promise before actually promise chaining. (Article is a variable not a promise)